.github/workflows/build.yaml

```diff
@@ -48,6 +48,14 @@ jobs:
               const fs = require('fs')
               const jsonString = fs.readFileSync('./package.json')
               var package = JSON.parse(jsonString)
+              var peerDeps = package.peerDependencies;
+              if(!peerDeps["@nebula.js/stardust"]) {
+                throw "Nebula peer dependency missing"
+              }
+              if(peerDeps["@nebula.js/stardust"].substr(0,1) !== ">") {
+                throw "Incorrect Nebula peer dependecy semver check, please use the form >=X.Y.Z and not ^X.Y.Z"
+              }
```

Comment:
Wondering if this is the correct place for it. Understand that it is the easiest place to check. But will it be clear enough when it fails?

Reply:
I think so. This is how it will look in the workflow run
![Screenshot 2024-04-10 at 15 16 38](https://github.com/qlik-oss/sn-gh-workflows/assets/968450/8a231e9a-e830-45d8-8f16-e96349e54939)

Reply:
This was from an old run. The error is now: `Incorrect Nebula peer dependecy semver check, please use the form >=X.Y.Z and not ^X.Y.Z`

Reply:
Oh yeah that is great

Reply:
This seems to cause issues for the container mono-repo (and the tables mono-repo as well from what I can see).
https://github.com/qlik-trial/nebula-containers/actions/runs/8688543433/job/23824396347

I can get the build to work by adding `stardust` as a peerDependencies to the nebula-containers package.json. Before we only had `stardust` as peerDeps for the actual chart packages inside the mono-repos.

Is that an okay fix or did this PR cause an unintended problem for the mono-repos? Ping @niekvanstaveren @enell

Reply:
I moved the v2 tag back to before this commit again. We are discussing how to resolve this. The above might actually be a good solution for now. I'll update both table and container repo if necessary